schema.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE IF EXISTS city_explorer;
+
+CREATE TABLE cities (
+  id SERIAL PRIMARY KEY,
+  search_query VARCHAR(255),
```

Comment:
If we add UNIQUE after VARCHAR(255) it'll only save one instance into the table!